### ISSUE TITLE
disable copy-to-clipboard on a per code block basis?

### ISSUE THREAD

**Issue by mmascolino** [matcornic/hugo-theme-learn#543]
is there any functionality to disable generation of the copy-to-clipboard image and functionality on a per code block basis?

**Comment by McShelby** [comment]
No. That's not implemented and I am not quite sure if this is even possible
